<?xml version="1.0"?>
<launch>

  <node pkg="srs_user_tests" name="octomap_loader" type="env_model_loader.py" output="screen" respawn="false">
    <param name="action" type="string" value="save"/>
    <param name="file" type="string" value="$(find srs_user_tests)/data/saved_octomap.enm"/>
  </node>

</launch>

